<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="confirmButtonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>640</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLineEdit" name="timeoutLineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="privateKeyLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="chatRecordLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="privateKeyLabel">
      <property name="text">
       <string>Private Key</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="maxRetriesLabel">
      <property name="text">
       <string>Max Retries(times)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="publicKeyLabel">
      <property name="text">
       <string>Public Key</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="maxRetriesLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="serverLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="serverLabel">
      <property name="text">
       <string>Server(host)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="publicKeyLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="timeoutLabel">
      <property name="text">
       <string>Timeout(seconds)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="chatRecordLabel">
      <property name="text">
       <string>Chat Record(path)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="trustedPeers">
      <property name="text">
       <string>Trusted Peers</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTableView" name="trustedPeersTableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
